app/src/main/res/values/strings.xml

```diff
@@ -102,6 +102,10 @@
     <string name="community_sortBy">Sort by</string>
     <string name="community_subscribe">Subscribe</string>
     <string name="community_users_month">%1$s users / month</string>
+    <string name="crashes">Crashes</string>
```

Comment:
Maybe "Crash logs", "All crash logs deleted", "Copy crash log" and "Delete all crash logs"?

Reply:
Yes that's much easier to read, changed 👍